### ISSUE TITLE
help wanted // javamail API not working in Android

### ISSUE THREAD

**Issue by sambhav2612** [sambhav2612/Android-EmailClient#1]
You can find the file under branch `patch-1`. Please help me finding the bug.

Please comment here so I can assign it to you on first come basis. Add yourself into CONTRIBUTORS.md under he following format: 

```
- <Your name>[<(@Your GitHub username)>](<Your profile link>)

For example: 

- Sambhav Jain [(@sambhav2612)](https://github.com/sambhav2612)
```

Thanks for stopping by.

**Comment by sathyapriya31** [comment]
@sambhav2612 what is the issue you currently facing? can you please explain

**Comment by sambhav2612** [comment]
@sathyapriya31 so I've put 3 fields in my XML with a button to send out email using javamail API. I've used my hardcoded credentials for Gmail login using SMTP protocol. (See the `GmailSender.java` file in source folder)

The problem here is that the pointer never reaches to that block where I'm sending the emails, because if it'd have the next statement should have printed a toast on my screen confirming my email to be sent.

**Comment by sathyapriya31** [comment]
@sambhav2612 We need to enable strict mode inorder to avoid android.os.NetworkOnMainThreadException while network access. If it's fine , then i can submit PR for this. Also we need to configure in Google Account ([https://myaccount.google.com/lesssecureapps?pli=1](https://myaccount.google.com/lesssecureapps?pli=1)) to allow less secure app aceess your account

**Comment by sambhav2612** [comment]
Yes, please submit the PR. And, I've turned allow less secure apps on.
